extensions/vault/old/ApacheHttpClientFactory.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.vault.internal.client;
```

Comment:
@vsevel, Can you please remove the `old` package ?

Reply:
I would have removed it just before the merge. but that's ok. done